### DISCUSSION TITLE
Build Failure: Ant BuildException in integration-arquillian-servers-auth-server-quarkus Module

### DISCUSSION THREAD

**Discussion by harsh2077** [keycloak/keycloak#34162] [Archive]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

dist/quarkus

### Describe the bug

I am encountering a build failure due to an Ant BuildException in the integration-arquillian-servers-auth-server-quarkus module of the Keycloak project. The build process fails during the execution of an Ant target, as indicated by the following Maven output:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  26:03 min
[INFO] Finished at: 2024-10-21T18:05:04+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (ant-generate-default) on project integration-arquillian-servers-auth-server-quarkus: An Ant BuildException has occured: The following error occurred while executing this line:
[ERROR] D:\New folder\keycloak\testsuite\integration-arquillian\servers\auth-server\quarkus\ant\configure.xml:10: exec returned: 1
[ERROR] around Ant part ...<ant antfile="ant/configure.xml" target="config">... @ 4:52 in D:\New folder\keycloak\testsuite\integration-arquillian\servers\auth-server\quarkus\target\antrun\build-main.xml
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :integration-arquillian-servers-auth-server-quarkus
```

### Version

26

### Regression

- [ ] The issue is a regression

### Expected behavior

The build should complete successfully without any test failures.

### Actual behavior

The build fails during the test phase with the following error message:
```
Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (ant-generate-default) on project integration-arquillian-servers-auth-server-quarkus: An Ant BuildException has occurred: The following error occurred while executing this line:
D:\New folder\keycloak\testsuite\integration-arquillian\servers\auth-server\quarkus\ant\configure.xml:10: exec returned: 1
around Ant part ...<ant antfile="ant/configure.xml" target="config">... @ 4:52 in D:\New folder\keycloak\testsuite\integration-arquillian\servers\auth-server\quarkus\target\antrun\build-main.xml
```

### How to Reproduce?

Run the Maven build with the following command:
`mvnw clean install -Pdistribution -DskipTests`

### Anything else?

I am a beginner in this so i am having trouble 
can u guide me in downloading and building keycloak

**Comment by shawkins** [comment]
Keycloak is creating an optimized default distribution for tests with this ant run. When it fails it usually means that some environment variables are being picked up by the build which make it not valid - check you environment variables for anything KC_...

I think adding -X to the mvnw will help you see more fully the error.

**Reply by harsh2077** [reply]
@shawkins 
hey i actually just need to run the login page for keycloak for customizing into the one i want 
if u know anything regarding this as well then pls tell 
i am not able to understand login page file of keycloak

**Reply by shawkins** [reply]
@harsh2077 please open a different discussion on theme / login page development.
